hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSet.java

```diff
@@ -113,13 +116,23 @@ public void testVolumeSetInitialization() throws Exception {
         "VolumeSet not initialized correctly");
     assertTrue(checkVolumeExistsInVolumeSet(volume2),
         "VolumeSet not initialized correctly");
+
+    MetricsRecordBuilder metricsRecords = getMetrics(volumeSet.getVolumeHealthMetrics());
+    assertGauge("TotalVolumes", 2, metricsRecords);
+    assertGauge("NumHealthyVolumes", 2, metricsRecords);
+    assertGauge("NumFailedVolumes", 0, metricsRecords);
```

Comment:
Can we consolidate this repeated 4 line pattern to a helper method like `assertNumVolumes(numHealthyVolumes, numFailedVolumes)` which makes sure that total volumes is equal to the sum of those?